### DISCUSSION TITLE
꾸준히 기록하기로 다짐한 나

### DISCUSSION THREAD

**Discussion by stopmin** [BDD-CLUB/challenge-every-week-posting#244] [5. 회고]
미리미리 기록하는 습관을 가지도록 결심했어요!! 개발 끝나고 쓰려고 했는데, 막상 써보니 개발하면서 써야 할 것 같네요. 오늘 [BDD 게시물](https://velog.io/@bdd14club/%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%EA%B8%B0%ED%9A%8D-2%EC%B0%A8-%EC%98%A8%EB%B3%B4%EB%94%A9-%ED%9A%8C%EA%B3%A0%EB%A1%9D)도 같이 작성하다보니 제 블로그 쓸 힘이 없군요...

앞으로는 개발하면서 동시에 쓰려고 합니다 :D
다음주에는 더 멋진 재밌는 글로 돌아오겠습니닷.

아래는 이번주 1주차 관련 게시물입니닷! 정말 별내용 없습니닷..

[초록스터디 1주차 게시물](https://velog.io/@stopmin/1%EC%A3%BC%EC%B0%A8-%EC%8A%A4%ED%94%84%EB%A7%81-%ED%94%8C%EB%A0%88%EC%9D%B4%EA%B7%B8%EB%9D%BC%EC%9A%B4%EB%93%9C-%EC%8A%A4%ED%84%B0%EB%94%94)

**Comment by shkisme** [comment]
!화!이!팅!
